docs/resources/resources-glossary.md

```diff
@@ -94,6 +94,12 @@ See also: [add-in commands](#add-in-commands).
 
 See also: [application](#application), [client](#client), [Office application, Office client](#office-application-office-client).
 
+## Long-Term Service Channel (LTSC)
+
+**LTSC** refers to the perpetual version of Office available through a volume-licensing agreement between Microsoft and your company.
+
+See also: [perpetual](#perpetual), [volume-licensed, volume-licensed perpetual, volume licensing](#volume-licensed-volume-licensed-perpetual-volume-licensing).
+
 ## Office application, Office client
 
 **Office client** refers to an Office application. The Office applications, or clients, that support Office Add-ins are Excel, OneNote, Outlook, PowerPoint, Project, and Word.
```

Comment:
Consider also noting in the VL section that LTSC could be another term and add link in See also

Reply:
Good catch. Added.